config/action.d/htaccess.conf

```diff
@@ -0,0 +1,59 @@
+# Fail2Ban configuration file
+# htaccess
+# You can ban IPs using the Deny directive of Apache in your htaccess. 
+# You should add, at the very end of the .htaccess  "Order Deny,Allow"
+# This action will append the "deny" rule at the end of the file.
+
+[Definition]
+
+# Option:  actionstart
+# Notes.:  command executed once at the start of Fail2Ban.
+# Values:  CMD
+#
+actionstart =
+
+
+# Option:  actionstop
+# Notes.:  command executed once at the end of Fail2Ban
+# Values:  CMD
+#
+actionstop =
+
+
+# Option:  actioncheck
+# Notes.:  command executed once before each actionban command
+# Values:  CMD
+#
+actioncheck =
+
+
+# Option:  actionban
+# Notes.:  command executed when banning an IP. Take care that the
+#          command is executed with Fail2Ban user rights.
+# Tags:    <ip>  IP address
+#          <file>  htaccess
+# Values:  CMD
+# You can have multiple virtual hosts
+#
+actionban = printf %%b "Deny from <ip>\n" >> "<htaccess1>"
+#	printf %%b "Deny from <ip>\n" >> "<htaccess2>"
```

Comment:
On a sober thought ;):
- rename htaccess to htaccess
- remove any htaccess2

since we can simply configure multiple actions per jail, thus allowing arbitrary number of htaccess files without need to modify this action